dataPortal/apache-atlas-ssl-redirect/container_files/etc/httpd/conf.d/virt.conf

```diff
@@ -0,0 +1,44 @@
+ServerName ENV_APACHE_SERVER_NAME 
+Listen 443
+
+<VirtualHost *:80>
+
+  ServerName ENV_APACHE_SERVER_NAME
+  RewriteEngine on
+  RewriteCond %{HTTPS}  off
+  RewriteRule ^/(.*)$   https://ENV_APACHE_SERVER_NAME/$1 [L,R]
+
+</VirtualHost>
+
+<VirtualHost *:443>
+
+  ServerName ENV_APACHE_SERVER_NAME
+
+  SSLEngine on
+  SSLProtocol +TLSv1.2
+  SSLHonorCipherOrder on
+  SSLCipherSuite HIGH:!aNULL:!MD5
+  SSLCertificateFile /certs/kpmp.crt
+  SSLCertificateKeyFile /certs/kpmp.key
+
+  RewriteEngine	on
+  RewriteRule	"^/endpoint/(.*?)$"	 "http://arranger-server.local:5050/endpoint/$1"	[P]
+  RedirectMatch ^/$ /repository/
```

Comment:
Will this redirect calls to arranger to /repository?

Reply:
Oh, probably doesn't matter since we'll get rid of this when we deploy the real thing...